### DISCUSSION TITLE
Job failed due to docker image not being in lowercase

### DISCUSSION THREAD

**Discussion by gaussianrecurrence** [community/community#26761] [Actions]
<p>I am trying to use a docker image from GitHub packages, which is under <a href="http://docker.pkg.github.com/%7Brepository%7D/" rel="noopener nofollow ugc">docker.pkg.github.com/{repository}/</a>:, but the problem is that the repository owner is an organization which name has an uppercase letter, and docker doesn’t accept image names with uppercase letters.<br>
I’ve tried to convert the repository name into lowercase and storing it in a env var, but .container.image doesn’t have access to env context.</p>
<p>I was wondering if someone encountered this issue and found a solution?<br>
I also thought of trying to define a function to convert an string to lowercase, but it seems it’s not possible.<br>
You can find an example of the issue here: <a href="https://github.com/Nordix/geode-native/runs/2421768124?check_suite_focus=true#step:2:23" class="inline-onebox" rel="noopener nofollow ugc">Test only linux · Nordix/geode-native@c80bece · GitHub</a></p>

**Comment by Simran-B** [comment, marked as answer]
<p>The <code>env</code> context is only available in steps, even though it can be set at workflow, job, and step level (also see <a href="https://github.community/t/how-to-use-env-context/16975" class="inline-onebox">How to use env context?</a>). There is another way to set a variable of sorts that should do the trick: Setting a job output and then refer to it with <code>${{ jobs.&lt;job_id&gt;.outputs.&lt;output&gt; }}</code> in another job.</p>
<pre><code class="lang-yaml">jobs:
  set-output:
    runs-on: ubuntu-latest
    outputs:
      image: ${{ steps.set.outputs.image }}
    steps:
      - name: Set output
        id: set
        shell: bash
        run: |
          name="Alpine"
          echo "::set-output name=image::${name,,}"
  use-output:
    runs-on: ubuntu-latest
    needs: set-output
    container:
      image: ${{ needs.set-output.outputs.image }}
    steps:
      - name: Print Linux OS info
        run: cat /etc/alpine-release
</code></pre>
<p>The <code>set-output</code> job runs a bit of Bash code to convert the string <code>Alpine</code> to lowercase and sets it as output. The <code>use-output</code> job depends on the <code>set-output</code> job and uses the output <code>image</code> to set the container.</p>

**Comment by gaussianrecurrence** [comment]
<p>Hi,</p>
<p>Thanks for the awnser. It actually worked as I was expecting.<br>
However I still expect that GH Actions developers should put in place a<br>
proper solution to this issue, as this seems to be a WA rather than a solution.</p>
<p>BR,<br>
Mario.</p>
